charts/reform-scan-blob-router/values.yaml

```diff
@@ -43,6 +43,7 @@ java:
 
     SMTP_HOST: "false"
     SEND_DAILY_REPORT_ENABLED: "false"
+    INCLUDE_WRITE_PERMISSION_IN_STORAGE_SAS_TOKEN: true
```

Comment:
do we want to make this change straight on prod or we want this disable for now and enable it for only demo ?

Reply:
`true` is what currently happens.